.github/workflows/blank.yml

```diff
@@ -52,28 +56,23 @@ jobs:
 
                   if (line.startsWith('+')) {
                     lineNumber++;
+                    changeLog += `Added (Line ${lineNumber}): ${line.substring(1)}\n`;
                     if (line.includes('TODO')) {
-                      try {
-                        await github.rest.pulls.createReviewComment({
-                          owner: context.repo.owner,
-                          repo: context.repo.repo,
-                          pull_number: context.issue.number,
-                          body: '변경된 라인에서 \'TODO\' 주석을 발견했습니다. 이 부분을 완료해주세요.',
-                          path: file.filename,
-                          commit_id: commit_id,
-                          position: position
-                        });
-                        console.log(`Comment added to ${file.filename} at position ${position}`);
-                      } catch (error) {
-                        console.error(`Error adding comment: ${error.message}`);
-                        console.error(`File: ${file.filename}, Position: ${position}`);
-                      }
+                      changeLog += `  - Note: 'TODO' comment found. Please address this.\n`;
                     }
-                  } else if (!line.startsWith('-')) {
+                  } else if (line.startsWith('-')) {
+                    changeLog += `Removed (Line ${lineNumber}): ${line.substring(1)}\n`;
+                  } else {
                     lineNumber++;
                   }
                 }
+                changeLog += '\n';
               }
+
+              fs.writeFileSync('change_log.txt', changeLog);
```

Comment:
3. Optimization: Writing to a file inside a loop can be inefficient. Consider aggregating all changes and writing them to the file once at the end to optimize performance.